dist/rpm/openQA.spec

```diff
@@ -425,6 +433,15 @@ install -m 644 contrib/munin/config/minion.config %{buildroot}/%{_sysconfdir}/mu
 install -m 755 contrib/munin/utils/munin-mail %{buildroot}/%{_datadir}/openqa/script/munin-mail
 %endif
 
+# anubis
+install -d -m 755 %{buildroot}%{_sysconfdir}/anubis
+install -d -m 755 %{buildroot}%{_sysconfdir}/anubis/data/cfg
+install -m 644 contrib/anubis/botPolicy.yaml %{buildroot}%{_sysconfdir}/anubis/data/cfg/botPolicy.yaml
+install -m 644 contrib/anubis/docker-compose.yml %{buildroot}%{_sysconfdir}/anubis/docker-compose.yml
+install -m 600 contrib/anubis/anubis.env %{buildroot}%{_sysconfdir}/anubis/anubis.env
+install -D -m 644 contrib/anubis/openqa-anubis.service %{buildroot}%{_unitdir}/openqa-anubis.service
+# install -m 644 contrib/anubis/nginx-anubis.conf %{buildroot}%{_sysconfdir}/anubis/nginx-anubis.conf
```

Comment:
why?

Reply:
i am not sure if and how i should add the nginx part. The nginx configuration is set up in [openQA/script/configure-web-proxy](https://github.com/os-autoinst/openQA/blob/8fc02990d191051d7fd201ea553ee3112c911efd/script/configure-web-proxy#L57), so maybe as a conditional there ? Also altering the existing configuration with `sed` commands or similar sounds rather fragile IMO, i guess best would be a dedicated anubis nginx template